dhis-2/dhis-api/src/main/java/org/hisp/dhis/programstagefilter/EventDataFilter.java

```diff
@@ -65,6 +65,9 @@ public class EventDataFilter implements Serializable {
   /** Like */
   private String like;
 
+  /** Exists */
+  private String ex;
```

Comment:
Why are we using a different operator name in the working list? Am I right that the operator is `null/!null` in our exporter API?